docs/source/user_guide/connector/info.ipynb

```diff
@@ -0,0 +1,114 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ```info()``` Function"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ```info()``` function provides guidelines for using Connector.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Parameters\n",
+    "\n",
+    "* ```config_path``` is the path to the folder containing configuration files. There are two ways to load configuration files. Details can be found in the previous configuraton file section.\n",
+    "\n",
+    "* ```update``` is used to specify if new configuration files should be pulled from the GitHub repo where up to date configuration files are hosted."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Response\n",
+    "\n",
+    "* ```Table``` displays table(s) of data that can be accessed.\n",
+    "* ```Parameters``` identifies parameters that can be used in the query function.\n",
+    "* ```Examples``` shows how methods Connector class can be called.\n",
+    "* ```Schema``` displays names and data types of attributes of the API response."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examples   "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dataprep.connector import info\n",
+    "info('yelp', update=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![title](assets/yelp_params_example.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![title](assets/yelp_schema_top.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![title](assets/yelp_schema_bottom.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After a connector object has been initialized with the ```connect``` function, info can also be called using the connector object:"
```

Comment:
And do we want to emphasize that calling info function this way does not need to include the config path as the parameter?